Resfeber/View Controllers/ExploreViewController.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  ExploreViewController.swift
+//  Resfeber
+//
+//  Created by David Wright on 11/8/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+class ExploreViewController: UIViewController {
+    
+    // MARK: - Properties
+    
+    fileprivate var collectionView: UICollectionView!
+    
+    fileprivate let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 28, weight: .bold)
+        label.textColor = .label
+        label.text = "Explore"
+        return label
+    }()
+    
+    fileprivate let profileButton: UIButton = {
+        let button = UIButton(type: .system)
+        let buttonDiameter: CGFloat = 32
+        button.setDimensions(height: buttonDiameter, width: buttonDiameter)
+        button.layer.cornerRadius = buttonDiameter / 2
+        button.layer.masksToBounds = true
+        button.contentMode = .scaleAspectFill
+        button.backgroundColor = .systemGray3
+        button.addTarget(self, action: #selector(profileImageTapped), for: .touchUpInside)
+        
+        let image = UIImage(systemName: "person.fill")!.withTintColor(.systemGray6, renderingMode: .alwaysOriginal)
+        button.setImage(image, for: .normal)
+        
+        return button
+    }()
+    
+    fileprivate let searchBar: UISearchBar = {
+        let sb = UISearchBar(frame: .zero)
+        sb.placeholder = "Search"
+        sb.searchBarStyle = .minimal
+        return sb
+    }()
+        
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureViews()
+    }
+    
+    // MARK: - Selectors
+    
+    @objc fileprivate func profileImageTapped() {
+        print("DEBUG: profileImageTapped..")
+    }
+    
+    // MARK: - Helpers
+    
+    fileprivate func configureViews() {
+        view.backgroundColor = UIColor.Resfeber.background
+        navigationController?.navigationBar.isHidden = true
+        
+        // Configure Title Label
+        view.addSubview(titleLabel)
+        titleLabel.anchor(top: view.safeAreaLayoutGuide.topAnchor, left: view.leftAnchor,
+                          right: view.rightAnchor, paddingTop: 4, paddingLeft: 20, paddingRight: 20)
```

Comment:
Spacing looks weird here

Reply:
@johnnydhicks A few lines, such as this one, are rather long so I chose to manually break it up across two lines. I know there are some style guides that have character limits for each line, although I am not following a strict limit here. What is your general recommendation for formatting long lines of code? Is it acceptable to just leave it all on one line and rely on Xcode's editor window auto wrap it?

Reply:
I think if you are going to do line breaks like this, I would do it at ever parameter, like this
`titleLabel.anchor(top: view.safeAreaLayoutGuide.topAnchor, 
                                left: view.leftAnchor,
                                right: view.rightAnchor, 
                                paddingTop: 4, 
                                paddingLeft: 20, 
                                paddingRight: 20)`

Reply:
Ok I like that. I'll make the change and submit another PR.